Oqtane.Server/Scripts/Tenant.1.0.1.sql

```diff
@@ -31,3 +31,9 @@ CREATE UNIQUE NONCLUSTERED INDEX IX_File ON [dbo].[File]
 	[Name]
 	) ON [PRIMARY]
 GO
+
```

Comment:
You have to create NEW script, not add to existing one. name it Tenant.1.0.1.NotificationFix.sql. this will not run on existing install